corgi/core/migrations/0098_load_missing_container_errata_builds.py

```diff
@@ -0,0 +1,63 @@
+from collections import defaultdict
+
+from django.db import migrations
+from django.db.models import Count
+
+from corgi.collectors.errata_tool import ErrataTool
+from corgi.tasks.errata_tool import slow_load_errata as current_slow_load_errata
+from corgi.tasks.prod_defs import brew_tag_to_et_prefixes
+from corgi.tasks.tagging import NO_MANIFEST_TAG
+
+
+def load_missing_container_errata_build(apps, schema_editor):
+    ProductStream = apps.get_model("core", "ProductStream")
+    CollectorErrataProductVersion = apps.get_model("collectors", "CollectorErrataProductVersion")
+
+    variants_by_stream = defaultdict(set)
+    # Allow this migration to be re-run in case of timeout or other failure
+    # Only get the streams we haven't processed yet
+    for stream in (
+        ProductStream.objects.annotate(num_components=Count("components"))
+        .filter(num_components__gt=0)
+        .exclude(tags__name=NO_MANIFEST_TAG)
+        .exclude(meta_attr__has_key="container_errata_checked")
+    ):
+        if stream.brew_tags:
+            for brew_tag in stream.brew_tags.keys():
+                trimmed_brew_tag, sans_container_released = brew_tag_to_et_prefixes(brew_tag)
+                variants = CollectorErrataProductVersion.objects.filter(
+                    brew_tags__overlap=[trimmed_brew_tag, sans_container_released]
+                ).values_list("variants__name", flat=True)
+                variants_by_stream[stream.name].update(variants)
+
+    et = ErrataTool()
+    for stream, variants in variants_by_stream.items():
+        stream_container_errata = et.get_errata_matching_variants(
+            list(variants), container_only=True
+        )
+        for errata in stream_container_errata:
+            current_slow_load_errata.delay(errata, force_process=True)
+        # Set these streams so they are not re-processed next run
+        checked_stream = ProductStream.objects.get(name=stream)
+        checked_stream.meta_attr["container_errata_checked"] = True
+        checked_stream.save()
+
+    # remove all the temporary container_errata_checked meta_attr keys
+    checked_streams = []
+    for stream in ProductStream.objects.filter(meta_attr__ship_checked=True):
```

Comment:
Is there a typo here?
```suggestion
    for stream in ProductStream.objects.filter(meta_attr__container_errata_checked=True):
```

Reply:
I have now removed this.